lib/model/emoji.dart

```diff
@@ -88,6 +88,60 @@ final class EmojiCandidate {
   }) : _aliases = aliases;
 }
 
+// Zulip's hand selected "popular" emojis, currently used as list of
+// quick emoji reactions available in the message context menu.
+// See: https://github.com/zulip/zulip/blob/3bad36ef8cf07cd57d0b257a739bae635a8527ac/web/shared/src/typeahead.ts#L22-L29
+const zulipPopularEmojis = [
+  EmojiCandidate(
+    emojiType: ReactionType.unicodeEmoji,
+    emojiCode: '1f44d',
+    emojiName: '+1',
+    aliases: ['thumbs_up', 'like'],
+    emojiDisplay: UnicodeEmojiDisplay(
+      emojiName: '+1',
+      emojiUnicode: '\u{1f44d}')), // '👍'
```

Comment:
I ended up providing this same data as part of #1112. Probably most convenient is if you cherry-pick the relevant commit:
bce508489 emoji: Add list of the "popular" emoji

into this PR.